docs/scripting-ref.rst

```diff
@@ -2230,3 +2230,126 @@ Trigger mouse release event in web page.
 **Async:**: no.
 
 See notes about mouse events in :ref:`splash-mouse-click`.
+
+.. _splash-send-keys:
+
+splash:send_keys
+----------------
+
+Send keyboard events to page context.
+
+**Signature:** ``splash:send_keys(keys)``
+
+**Parameters**
+
+* keys - string representing the keys to be sent as keyboard events.
+
+**Returns:** nil
+
+**Async:**: no.
+
+Key sequences are specified by using a small subset of emacs edmacro syntax:
+
+* whitespace is ignored and only used to separate the different keys
+* characters are literally represented
+* words within brackets represent function keys, like ``<Return>``, ``<Left>``
+  or ``<Home>``. See `Qt docs`__ for a full list of function keys. ``<Foo>``
+  will try to match ``Qt::Key_Foo``.
+
+__ http://doc.qt.io/qt-5/qt.html#Key-enum
+
+Following table shows some examples of macros and what they would generate on
+an input:
+
+======================    ===============
+Macro                     Result
+======================    ===============
+``Hello World``           ``HelloWorld``
+``Hello <Space> World``   ``Hello World``
+``< S p a c e >``         ``<Space>``
+``Hello <Home> DEL``      ``ello``
+``Hello <Backspace>``     ``Hell``
+======================    ===============
+
+Key events are not propagated immediately until event loop regains control,
+thus :ref:`splash-wait` must be called to reflect the events.
+
+.. _Qt key-enum: http://doc.qt.io/qt-5/qt.html#Key-enum
+
+.. _splash-send-text:
+
+splash:send_text
+----------------
+
+Send text as keyboard events to page context.
+
+**Signature:** ``splash:send_text(text)``
+
+**Parameters:**
+
+* text - string to be sent as keyboard events.
```

Comment:
In `send_text` doc I'd probably avoid mentioning keyboard events altogether and just say that the text is sent as inputu literally, character-for-character, and that's all. And refer to `send_keys` as something that works closer to events and function/special keys.